src/esn.calendar.libs/app/components/time-select/time-select.controller.js

```diff
@@ -29,6 +29,7 @@ function calTimeSelectController($scope, $element, calMoment) {
   }
 
   function onInputKeydown(event, mdMenu) {
+    if (event.key === 'Tab') return mdMenu.close();
```

Comment:
People will break you awesome improvement if there is no test for it ;)

Reply:
done, thanks